code/game/objects/items/weapons/batons.dm

```diff
@@ -66,6 +66,9 @@
   * * user - The attacking user
   */
 /obj/item/melee/classic_baton/proc/baton_knockdown(mob/living/target, mob/living/user)
+	if(user.mind?.martial_art?.no_baton)
+		to_chat(user, "<span class='warning'>The baton feels off balance in your hand due to your judo training!</span>")
+		return
```

Comment:
the text is specific to having judo, but the check isnt specific to judo.

if we for instance set this to TRUE on carp the text wouldnt be correct. I suggest you add a `no_baton_reason` var on martial arts and use that here.